python/core/auto_generated/geometry/qgsgeometryutils.sip.in

```diff
@@ -424,8 +424,8 @@ angular deviation (in radians) allowed when testing for regular point spacing.
 
     static int segmentSide( const QgsPoint &pt1, const QgsPoint &pt3, const QgsPoint &pt2 ) /HoldGIL/;
```

Comment:
wouldn't it be simpler to rename the arg to match the description? I don't know if your changes are bassed on the argument order or the current naming.

Reply:
We can't -- it's an API break (eg if a python script is calling this method using named parameters)